caffe2/share/contrib/benchmark_harness/platforms/host/host_platform.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+from platforms.platform_base import PlatformBase
+import subprocess
+import platform
+from utils.arg_parse import getArgs, getParser
+from utils.custom_logger import getLogger
+
+getParser().add_argument("--host", action="store_true",
+    help="Run the benchmark on the host.")
+
+class HostPlatform(PlatformBase):
+    def __init__(self):
+        super(HostPlatform, self).__init__()
+        self.output = None
+
+    def setupPlatform(self):
+        pass
+
+    def runBenchmark(self):
+        cmd = [
+            "--logtostderr", "1",
+            "--init_net", getArgs().init_net,
+            "--net", getArgs().net,
+            "--input", getArgs().input,
+            "--warmup", str(getArgs().warmup),
+            "--iter", str(getArgs().iter),
+            ]
+        if getArgs().program:
+            cmd.insert(0, getArgs().program)
+        elif getArgs().exec_base_dir:
+            cmd.insert(0, getArgs().exec_base_dir +
+                '/build/caffe2/share/contrib/binaries/caffe2_benchmark/binaries/caffe2_benchmark')
```

Comment:
this seems a bit fragile, is there any way to make it more robust to future changes?